skipper.go

```diff
@@ -802,6 +811,20 @@ func createDataClients(o Options, auth innkeeper.Authentication) ([]routing.Data
 		}
 	}
 
+	if o.RemoteRoutesFile != "" {
+		for _, rf := range strings.Split(o.RemoteRoutesFile, ",") {
+			client, err := eskipfile.RemoteWatch(&eskipfile.RemoteWatchOptions{
+				RemoteFile:    rf,
+				FailOnStartup: o.RemoteRoutesFailOnStartup,
```

Comment:
Do we need this configurable? Maybe just always fail like for routes from file?

Reply:
I think this configurable makes sense from reliability perspective. I would say failing to load on startup does not really always imply a long term failure of the remote service (which can succeed just on a subsequent try) and it might be multiple opinions how to behave on such an occasion.